nx_parallel/algorithms/approximation/connectivity.py

```diff
@@ -5,18 +5,25 @@
 import nx_parallel as nxp
 
 __all__ = [
-    "all_pairs_node_connectivity",
+    "approximate_all_pairs_node_connectivity",
 ]
 
 
-def all_pairs_node_connectivity(G, nbunch=None, cutoff=None, get_chunks="chunks"):
+def approximate_all_pairs_node_connectivity(
+    G, nbunch=None, cutoff=None, get_chunks="chunks"
+):
     """The parallel implementation first divides the a list of all permutation (in case
     of directed graphs) and combinations (in case of undirected graphs) of `nbunch`
     into chunks and then creates a generator to lazily compute the local node
     connectivities for each chunk, and then employs joblib's `Parallel` function to
     execute these computations in parallel across all available CPU cores. At the end,
     the results are aggregated into a single dictionary and returned.
 
+    Note, this function uses the name `approximate_all_pairs_node_connectivity` while
```

Comment:
also, added this to make the user aware that this function uses a different name while dispatching.

Reply:
I thought that users would never call this function directly -- that is, they would call the networkx function which in turn would call the appropriate function for them.  Do I understand the public facing api correctly?

Reply:
I don't think we are that strict about how someone "should" use a backend and a backend is also a package. So, I added this for someone who might want to call nx_parallel's implementation directly for their graph. But, as mentioned in the note added in README(in this PR) it's not the recommended way. If you think it's too much info to put in the main docs, then let me know, I can remove it and just keep(or maybe change a little) the first note in README.

Reply:
OK... and I have a question:  in the README file just after the paragraph describing this, the examples says:
```
 nx.approximate_all_pairs_node_connectivity(H)  # runs the parallel implementation in `approximation/connectivity`
```
But is this right?  `nx.approximate_all_pairs_node_connectivity(H)` will raise due to the function not being found.  In `nx` we call it as `nx.approximation.all_pairs_node_connectivity(H)`

I was thinking that the dispatcher calls the backend using the function `nx_parallel.approximate_all_pairs_node_connectivity(H)`.  but the user would still call it after importing networkx as `nx.approximation.all_pairs_node_connectivity(H)`.

If I am right, we could allow both in the nx-parallel package (if you want to allow that) by including both routes to the name in the `__init__.py` files. But for now just having the longer name is Ok too. So I'm Ok with what is in this PR...  But could you check the README.md example for how to call those functions?

Reply:
Thank you very much for pointing this out!
fixed in https://github.com/networkx/nx-parallel/pull/58